pages/tutorials/migrating_from_bamboo.md.erb

```diff
@@ -64,7 +64,7 @@ steps:
     artifact_paths: "build/*"
 
   # To prevent the "make test" stage from running before "make" has finished,
-  # we seperate the command with a "wait" step.
+  # we separate the command with a "wait" step.
```

Comment:
My spelling mistake pre-commit hook picked this up. Let me know if you'd like me to cherry pick into another PR.

Reply:
Rad, thanks! It'll be fine in with the rest 😊